saleor/graphql/discount/mutations.py

```diff
@@ -366,7 +380,7 @@ def perform_mutation(cls, _root, info, **data):
             info, data.get("id"), only_type=Sale, field="sale_id"
         )
         cls.add_catalogues_to_node(sale, data.get("input"))
-        return SaleAddCatalogues(sale=sale)
+        return SaleAddCatalogues(sale=ChannelContext(node=sale, channel_slug=None))
```

Comment:
Can we restore the old return? 
We adding ChannelContext in `succesResponse` in `SaleUpdateDiscountedPriceMixin`

Reply:
`SaleAddCatalogues ` and `SaleRemoveCatalogues` itherit from `BaseDiscountCatalogueMutation(BaseMutation)`, we can use `success_response ` only with `ModelMutation`.

`SaleUpdateDiscountedPriceMixin ` is related with other mutations:

- SaleCreate

- SaleUpdate

- SaleDelete